crt_portal/cts_forms/views.py

```diff
@@ -287,6 +288,9 @@ def index_view(request):
 
     all_args_encoded = urllib.parse.quote(f'{page_args}&page={page}')
 
+    emails_in_reports_on_this_page = [r.contact_email for r in requested_reports]
```

Comment:
May be slightly more efficient to do `requested_reports.values_list('contact_email', flat=True)`? Wonder if this will help the ORM with optimizing the aggregation query.